tests/dummy/config/targets.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-env node */
 module.exports = {
   browsers: [
-    'ie 9',
```

Comment:
is this necessary?

Reply:
This is just to run our test suite with native async/await. Without that change, it would get transpiled, but then it failes because of the missing regenerator runtime. As we are testing in Chrome anyways, I though to just use the native implementation. FWIW, ember-qunit does [the same](https://github.com/emberjs/ember-qunit/blob/master/tests/dummy/config/targets.js)

Reply:
yeah, but changing the support matrix would need a major version bump and I'd like to avoid that for now.

we can't use async/await in this lib because then we would require that our users also support it natively or via regenerator. we can use it in the test suite (of the test suite 🙈), but then we should use `ember-maybe-import-regenerator-for-testing` and the corresponding linting to make sure that we definitely do not use it in prod code.

Reply:
I added async/await only to the test suite, so users should not be affected by this! And this is the `targets.js` file of the dummy app, so only affecting our test builds, not that of users. So I *think* that should be fine? 

I agree we have to make sure linting catches accidental use of async/await in the lib code itself, I will check this...

Reply:
I wanted to write that removing it would prevent us from catching IE9 bugs in CI, but then I realized how much bullshit that was... 😆 

seems fine, thanks

Reply:
Yes, unless you want to enter a world of pain, and use saucelabs, that would not really work! 😉
(*heard browserstack would be better, but never tried it*)